docs/src/embeddings/default_embedding_functions.md

```diff
@@ -151,28 +151,31 @@ Allows you to set parameters when registering a `sentence-transformers` object.
     You can also load many other model architectures from the library. For example models from sources such as BAAI, nomic, salesforce research, etc.
     See this HF hub page for all [supported models](https://huggingface.co/models?library=sentence-transformers).
 
-Visit sentence-transformers [HuggingFace HUB](https://huggingface.co/sentence-transformers) page for more information on the available models.
-```python
-db = lancedb.connect("/tmp/db")
-registry = EmbeddingFunctionRegistry.get_instance()
-func = registry.get("sentence-transformers").create(device="cpu")
-
-class Words(LanceModel):
-    text: str = func.SourceField()
-    vector: Vector(func.ndims()) = func.VectorField()
+!!! note "BAAI Embeddings example"
+    Here is an example that uses BAAI embedding model from the HuggingFace Hub [supported models](https://huggingface.co/models?library=sentence-transformers)
+    ```python
+    db = lancedb.connect("/tmp/db")
+    registry = EmbeddingFunctionRegistry.get_instance()
+    func = registry.get("sentence-transformers").create(name="BAAI/bge-small-en-v1.5", device="cpu")
+
```

Comment:
can use the term `model`. i know it's embeddingfunction but it's still more intuitive than `func`